governance.md

```diff
@@ -0,0 +1,164 @@
+
+To stand up this Governance for a 1.0 good-enough minimal-viable-governance (MVG), we are looking to our key predecessors for some of our starting points.
+As an initial set of principles we are combining elements of the [Kubernetes community principles](https://github.com/kubernetes/community/blob/master/governance.md#principles) and the [OpenInfra Foundations Four Opens](https://openinfra.dev/four-opens/).
+
+#### Op1st Community Principles 1.0
+Our mission requires us to use open source principles in ways beyond licensing the code we write, but at the core we are an open source community and want to adhere to the best possible principles for creating an open, welcoming, and inclusive community.
+
+These are the Op1st community principles we adhere to:
+1. **Open Source** : Any content we create is released under an open source license. Content includes software, documentations, configuration files, and training materials.
+  - This is crucial to fulfilling our mission around open sourcing operations
+2. **Welcoming and respectful** : We intend to be an open, welcoming, and inclusive community.
+Diversity is our strength.
+We have and follow a Code of Conduct.
+  - We currently follow the OpenInfra Foundation Code of Conduct as a member of the OpenInfra Labs.
+3. **Open Design** : We follow a transparent and open process for planning and designing any content, services, and features of Op1st.
+Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles.
+4. **Open Work** : Two of our core tools for successful community building are transparency and accessibility.
+To support that, work and collaboration must be done in public to the greatest degree possible.
+Refer to the section on Community Groups and SIGs in particular for details on how this is performed.
+5. **Truly Open Community** : The best way to build a sustainable open source community is to ensure the community is a cohesive, inclusive, and equitable place where all the voices are heard and anyone can rise to leadership positions.
+
+### Code of Conduct
+
+We are using the [upstream OpenInfra Foundation Code of Conduct](https://openinfra.dev/legal/code-of-conduct) until we can convene the new Community SIG to lead an open effort to create our own, modern code of conduct.
+
+### Community membership
+**Note:** This document is a work in progress, especially these membership guidelines
+
+We are opening the Governance 1.0 directly borrowing from the [Kubernetes community group definitions](https://github.com/kubernetes/community/blob/master/community-membership.md), then reviewing and refining for the Governance 1.1.
+
+This doc outlines the various responsibilities of contributor roles in Op1st.
+The Op1st project is subdivided into subprojects under SIGs.
```

Comment:
s/project/community/

Reply:
The Operate First community is organized primarily into Special Interest Groups, or SIGs. Specific work efforts within SIGs are divided into subprojects. 

https://github.com/kubernetes/community/blob/master/governance.md#sigs
https://github.com/kubernetes/community/blob/master/governance.md#subprojects

Reply:
Discussed and resolved in [this comment above](https://github.com/operate-first/community/pull/107/files/092e770de008cf2c8ccfd95f528b3326618d2baa#r770799637); if there is disagreement or clarification, let's keep it in that thread.